pulp_smash/pulp_smash_internals.ipynb

```diff
@@ -0,0 +1,989 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pulp-Smash Internals"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Why?\n",
+    "\n",
+    "Get familiar with Pulp-Smash, and get to know how Pulp is tested. "
```

Comment:
```suggestion
    "Get familiar with Pulp-Smash and get to know how Pulp is tested. "
```
Comma is not required . Called an Oxford comma. Some people like it. Most modern people complain. Since it is not a list of 3 or more items, it is generally not left.

https://en.wikipedia.org/wiki/Serial_comma